playground/src/import-url/import-url.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import './import-url.scss';
+
+import events from '../events/events.js';
+import * as Button from '../button/button.js';
+import * as Document from '../document/document.js';
+import FlyIn from '../fly-in/base.js';
+
+export const EVENT_REQUEST_URL_CONTENT = 'event-request-url-content';
+export const EVENT_UPDATE_EDITOR_CONTENT = 'event-update-editor-content';
+
+/* eslint-disable max-len */
+const URL_VALIDATION_REGEX = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/gm;
+
+export function createImportURLView(target, trigger) {
+  return new ImportURL(target, trigger);
+}
+
+class ImportURL extends FlyIn {
+  constructor(target, trigger) {
+    super(target);
+
+    this.target = target;
+    this.trigger = Button.from(trigger, this.toggle.bind(this));
+
+    const content = document.createElement('div');
+    content.className = 'import-url';
+    content.innerHTML = `
```

Comment:
Just noticed this: the webpack setup actually has a handlebars loader configured. See the loader for inspiration:

https://github.com/ampproject/amp.dev/blob/b53080d0d04f5c393b9e2625a4787db4b657d3ec/playground/src/loader/base.js#L61

https://github.com/ampproject/amp.dev/blob/b53080d0d04f5c393b9e2625a4787db4b657d3ec/playground/src/loader/base.js#L30

Moving all of the markup to actual templates would be great.